scripts/openmc-make-compton

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import tarfile
+from urllib.request import urlopen
+
+import numpy as np
+import h5py
+
+
+base_url = 'http://geant4.cern.ch/support/source/'
+filename = 'G4EMLOW.6.48.tar.gz'
+block_size = 16384
+
+# ==============================================================================
+# DOWNLOAD FILES FROM GEANT4 SITE
+
+# Establish connection to URL
+req = urlopen(base_url + filename)
+
+# Get file size from header
+file_size = req.length
+downloaded = 0
+
+# Check if file already downloaded
+download = True
+if os.path.exists(filename):
+    if os.path.getsize(filename) == file_size:
+        print('Already downloaded ' + filename)
+        download = False
+    else:
+        overwrite = input('Overwrite {}? ([y]/n) '.format(filename))
+        if overwrite.lower().startswith('n'):
+            download = False
+
+if download:
+    # Copy file to disk
+    print('Downloading {}... '.format(filename), end='')
+    with open(filename, 'wb') as fh:
+        while True:
+            chunk = req.read(block_size)
+            if not chunk: break
```

Comment:
Same comment as above re: PEP8 compliance with multiple statements on the same line.